README.md

```diff
@@ -144,6 +144,17 @@ Lookups and indexes are not implemented yet
 ([bug #9](https://github.com/danni/django-postgres-composite-types/issues/9),
 [bug #10](https://github.com/danni/django-postgres-composite-types/issues/10)).
 
+Running Tests
+-------------------
+Clone the repository, go to it's base directory and run the following commands.
+
+    pip install -r requirements.in
+    pip install -r test_requirements.in
+    export PYTHONPATH="$(pwd)"
+    export DJANGO_SETTINGS_MODULE='tests.settings'
+    django-admin test
```

Comment:
I don't have any experience with `tox`. Please, correct it yourself.

Reply:
You can copy this directly out of the Travis config: https://github.com/danni/django-postgres-composite-types/blob/master/.travis.yml#L22

https://tox.readthedocs.io/en/latest/

```
pip install tox
tox
```

Or if you want a specific environment

```
tox -e py37-dj2.0
```

While you're at it. Since you're now adding code for Django 2.1, add that to Travis and Tox.ini